examples/custom.rs

```diff
@@ -124,6 +125,10 @@ pub enum Error {
     InvalidStringFormat,
 }
 
+impl From<Infallible> for Error {
+    fn from(never: Infallible) -> Self { match never {} }
+}
```

Comment:
I think examples should be exempt from adding things they don't use but I won't block this.